app/models/review.rb

```diff
@@ -0,0 +1,30 @@
+# == Schema Information
+#
+# Table name: reviews
+#
+#  id            :integer          not null, primary key
+#  reviewer_name :string
+#  rating        :integer
+#  comment       :text
+#  restaurant_id :integer          not null
+#  created_at    :datetime         not null
+#  updated_at    :datetime         not null
+#
+
+class Review < ApplicationRecord
+
+  validates_presence_of :rating, :reviewer_name
+  validates_numericality_of :rating, greater_than_or_equal_to: 0, less_than_or_equal_to: 3,
+                            message: 'Rating value should be between 1 to 3'
```

Comment:
There's a slightly newer style for writing this:
```ruby
validates :foo, presence: true
validates :bar, numericality: { greater_than_or_equal: 0, ... }
```

For more: https://apidock.com/rails/ActiveModel/Validations/ClassMethods/validates

Reply:
Thanks!